src/verify/VerificationStartScreen.tsx

```diff
@@ -81,7 +82,11 @@ function VerificationStartScreen({
   const cachedNumber = useSelector(e164NumberSelector)
   const cachedCountryCallingCode = useSelector(defaultCountryCodeSelector)
   const walletAddress = useSelector(walletAddressSelector)
-  const { step, totalSteps } = useSelector(registrationStepsSelector)
+  const onboardingProps = useTypedSelector(onboardingPropsSelector)
+  const { step, totalSteps } = getOnboardingStepValues(
+    Screens.VerificationStartScreen,
+    onboardingProps
+  )
```

Comment:
looks like there is still some bespoke logic for determining the next onboarding screen to show from this one-- not ideal

Reply:
it seems like the problem is that there are 2 screens you might want to show next-- OnboardingSuccessScreen if they successfully verify, or home if they skip. Two ideas for handling this branching:
- treat "home" as the next screen from this one, and pass a callback to OnboardingSuccessScreen as a prop that takes it to the "next screen in onboarding" (always home)
- put the branching logic inside `goToNextOnboardingScreen`, and choose the next screen by checking whether a user has verified their PN

Reply:
The way I think about it is VerificationStartScreen, VerificationInputScreen, and OnboardingSuccessScreen are all part of the Verification Step, and the logic for navigation between those screens should be handled internally since the logic is tightly coupled. 

The functions like `goToNextOnboardingScreen` should only be used to navigate between loosely coupled steps in the onboarding flow. Maybe I should change the name to `goToNextOnboardingStep` and create a new `OnboardingSteps` enum to capture the difference between screens and steps in this case.